tests/test_prod_info.py

```diff
@@ -205,6 +205,52 @@ def test_deserialize_update_prod_info(self):
         prod_info = ProductInformation(prod_name = "ert", new_qty = -1, used_qty = -1, open_boxed_qty = -1, restock_level = -2, restock_amt = -1)
         self.assertRaises(DataValidationError, prod_info.deserialize_update, data)
 
+    def test_query_by_prod_name(self):
```

Comment:
Naming issue. test_find_by_... to match tested function names.
line 222, 238 as well.